DOC_1773: full-featured.adoc (#2489)
tinymce/tinymce-docs: modules/ROOT/pages/demo/full-featured.adoc

[[fullfeaturedincludingpremiumplugins]]
== Full Featured: Including Premium Plugins

> IMPORTANT: This demonstration page is for {productname} 5.10. {productname} 5.10 is still a supported version but it is not the current release. The full-featured demonstration page for {productname} 6 is xref:tinymce/6/full-featured-premium-demo.adoc[available] in the {productname} 6 xref:tinymce/6/[documentation].

Reviewer: We probably shouldn't be adding this to a single page... if anything we should be adding it to all demo pages. I know at one point there was discussions about making it should a banner on every 5.x docs page. I also don't think we should be mentioning the minor version here as nowhere else in the entire docs do we do this type of thing. You also cannot use an xref for something that's not part of the build, so those need to be links. Lastly, we should be using the asciidoc variables instead of hardcoding this.

```suggestion
IMPORTANT: This demonstration page is for {productname} {productmajorversion}. {productname} {productmajorversion} is still a supported version but it is not the current release. The full-featured demonstration page for {productname} 6 is link:{{site-url}}/tinymce/6/full-featured-premium-demo/[available] in the {productname} 6 link:{{site-url}}/tinymce/6/[documentation].
```

Author: > We probably shouldn't be adding this to a single page

FWIW, I agree. I was dealing with the ticket as presented. Apparently this page, in particular, gets a fair chunk of traffic that leads to conversions. Hence the focus on getting this page updated to draw attention to the equivalent page in the 6.x docs.

> if anything we should be adding it to all demo pages.

Perhaps a ticket for taking the skeleton added here and turning into a `/partial/` would be the ticket (pun fully intended).

Reviewer: > FWIW, I agree. I was dealing with the ticket as presented.

Hmm... we said we didn't want this in the #dx, so I'm guessing the Jira was logged incorrectly.

Author: > incorrectly

s/the Jira was logged incorrectly./the Jira was logged ‘incorrectly.’/

/me ::wonders if those text-based air-quotes will be parsed as intended.::

Author: Re _adding it to all demo pages_: [DOC-1778](https://ephocks.atlassian.net/browse/DOC-1778) now exists.

Reviewer: @brian-forte that doesn't solve this though... we cannot have a single page have this as that's very confusing and gives us an unprofessional appearance. Additionally, as I noted above adding this was not what was agreed to previously, so it should be removed from this PR and we can follow up on adding it properly later via the Jira you mentioned.

* xref:plugins/premium/moxiemanager.adoc[MoxieManager (Premium Plugin)] -- xref:plugins/premium/tinydrive.adoc[*{cloudfilemanager}*] included instead.
* xref:plugins/opensource/autoresize.adoc[Autoresize] -- Resizes the editor to fit the content.

> * xref:plugins/opensource/bbcode.adoc[BBCode] -- Changes the markup used for the content. xref:release-notes/6.0-upcoming-changes/#plugins[Deprecated] as of {productname} 5.9.

Reviewer 2: ```suggestion
* xref:plugins/opensource/bbcode.adoc[BBCode] -- Changes the markup used for the content. xref:release-notes/6.0-upcoming-changes.adoc#plugins[Deprecated] as of {productname} 5.9.
```
This link is broken like this, this should fix it as it has on my local build.

* xref:plugins/opensource/code.adoc[Code] -- xref:plugins/premium/advcode.adoc[*Advanced Code Editor*] included instead.

> * xref:plugins/opensource/fullpage.adoc[Full Page] -- Used for modifying HTML `<head>` elements. xref:release-notes/6.0-upcoming-changes/#plugins[Deprecated] as of {productname} 5.9.

Reviewer: All these links still need to be fixed as per @MitchCTiny's previous comment.

Author: All these links now fixed as per @MitchCTiny’s previous comment.

> * xref:plugins/opensource/fullpage.adoc[Full Page] -- Used for modifying HTML `<head>` elements. xref:release-notes/6.0-upcoming-changes.adoc/#plugins[Deprecated] as of {productname} 5.9.

Reviewer 2: ```suggestion
* xref:plugins/opensource/fullpage.adoc[Full Page] -- Used for modifying HTML `<head>` elements. xref:release-notes/6.0-upcoming-changes.adoc#plugins[Deprecated] as of {productname} 5.9.
```

* link:{plugindirectory}[All premium plugins].
* xref:plugins/opensource/autoresize.adoc[Autoresize] -- Resizes the editor to fit the content.

> * xref:plugins/opensource/bbcode.adoc[BBCode] -- Changes the markup used for the content. xref:release-notes/6.0-upcoming-changes.adoc/#plugins[Deprecated] as of {productname} 5.9.

Reviewer 2: ```suggestion
* xref:plugins/opensource/bbcode.adoc[BBCode] -- Changes the markup used for the content. xref:release-notes/6.0-upcoming-changes.adoc#plugins[Deprecated] as of {productname} 5.9.
```

> * xref:plugins/opensource/fullpage.adoc[Full Page] -- Used for modifying HTML `<head>` elements. xref:release-notes/6.0-upcoming-changes.adoc/#plugins[Deprecated] as of {productname} 5.9.

Reviewer 2: ```suggestion
* xref:plugins/opensource/fullpage.adoc[Full Page] -- Used for modifying HTML `<head>` elements. xref:release-notes/6.0-upcoming-changes.adoc#plugins[Deprecated] as of {productname} 5.9.
```

> * xref:plugins/opensource/spellchecker.adoc[Spellchecker] -- Requires Server-Side components, see: xref:general-configuration-guide/spell-checking.adoc[Check spelling in {productname}]. xref:release-notes/release-notes54.adoc/#thefreetinymcespellcheckerplugin[Deprecated] as of {productname} 5.4.

Reviewer 2: ```suggestion
* xref:plugins/opensource/spellchecker.adoc[Spellchecker] -- Requires Server-Side components, see: xref:general-configuration-guide/spell-checking.adoc[Check spelling in {productname}]. xref:release-notes/release-notes54.adoc#thefreetinymcespellcheckerplugin[Deprecated] as of {productname} 5.4.
```
